schemas/com.olark/transcript/jsonschema/1-0-0

```diff
@@ -0,0 +1,210 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for an Olark transcript event",
+  "self": {
+    "vendor": "com.olark",
+    "name": "transcript",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "tags": {
+      "type": "array",
+      "items": {
+        "type": "string",
+        "maxLength": 128
+      }
+    },
+    "items": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "body": {
+            "type": "string"
+          },
+          "timestamp": {
+            "type": "string",
+            "maxLength": 64
```

Comment:
Do we know what format this timestamp is in? It has a maxLength.

Reply:
@alexanderdean It appears to be in float unfortunately e.g. "timestamp":"1307116657.1" or "timestamp":"1307116661.25"

Reply:
Can we translate to ISO8601?

Reply:
@alexanderdean That is already done (https://github.com/snowplow/iglu-central/pull/664/commits/7d0a8972e654644d8c3bf6731f1dcaf51ae3e5f1#diff-29edb5b9239c6b0b35b397d86d5d0cd1R27)